frontend/src/libs/components/card/card.tsx

```diff
@@ -12,6 +12,8 @@ type Properties = {
   isActive?: boolean;
   iconName?: IconName;
   iconColor?: ValueOf<typeof IconColor>;
+  iconWidth?: number;
+  iconHeight?: number;
```

Comment:
I believe we should have the same icon width and height for all card components, no need to pass them via props
```suggestion
```